lua/plenary/vararg/rotate.lua

```diff
@@ -0,0 +1,94 @@
+-- dont edit this file, it was generated
```

Comment:
I think this file should be generated like:

```
local rotate = {}
rotate[2] = function(A0, A1) ... end
rotate[3] = ...
```

and instead of a bunch of if/then statements, just use a look up and if it's nil then do the default case.